src/v/resource_mgmt/storage.cc

```diff
@@ -643,4 +664,69 @@ ss::future<> disk_space_manager::manage_data_disk(uint64_t target_size) {
     co_await _storage->invoke_on_all([](api& api) { api.trigger_gc(); });
 }
 
+disk_space_manager::probe::probe(disk_space_manager* sm)
+  : _sm(sm) {}
+
+void disk_space_manager::probe::setup_metrics() {
+    if (config::shard_local_cfg().disable_metrics()) {
+        return;
+    }
+
+    namespace sm = ss::metrics;
+
+    const auto group_name = prometheus_sanitize::metrics_name(
+      "space_management");
+
+    std::vector<sm::impl::metric_definition_impl> defs;
+
+    /*
+     * The currently configured target size for the data disk. When disk
+     * usage exceeds this amount then the expectation is that space
+     * management will reclaim data.
+     *
+     * If space management is disabled the metric value is 0.
+     */
+    defs.emplace_back(sm::make_gauge(
+      "target_disk_size",
+      [this]() { return _sm->enabled() ? _sm->_target_size : 0; },
+      sm::description("Target maximum number of stored bytes.")));
+
+    defs.emplace_back(sm::make_gauge(
+      "total_disk_usage",
+      [this]() { return _total_usage; },
+      sm::description(
+        "Total amount of disk usage under control of space management.")));
+
+    defs.emplace_back(sm::make_gauge(
+      "total_retention_reclaimable",
+      [this]() { return _retention_reclaimable; },
+      sm::description("Total amount of reclaimable data through standard "
+                      "retention policy.")));
+
+    defs.emplace_back(sm::make_gauge(
+      "total_available_reclaimable",
+      [this]() { return _available_reclaimable; },
+      sm::description("Total amount of available reclaimable data by space "
+                      "management.")));
+
+    defs.emplace_back(sm::make_gauge(
+      "total_local_retention_reclaimable",
+      [this]() { return _local_retention_reclaimable; },
+      sm::description("Total amount of reclaimable data above the local "
+                      "retention target.")));
+
+    defs.emplace_back(sm::make_gauge(
+      "target_excess",
+      [this]() { return _target_excess; },
+      sm::description("Amount of data usage that exceeds target threshold.")));
+
+    defs.emplace_back(sm::make_gauge(
+      "reclaim_estimate",
+      [this]() { return _reclaim_estimate; },
+      sm::description("Estimated amount of data reclaimed by space "
```

Comment:
`Estimated amount of data reclaimed by space`

technically, "... data to be reclaimed ..." describes the metric better as it is computed prior doing any reclamation

Reply:
fixed